File: algorithm/java/Q_260729/MissingSum_260729.java
//문제: 없는 숫자 더하기
//링크: (https://school.programmers.co.kr/learn/courses/30/lessons/86051?language=python3)
//날짜: 2026-07-29

import java.util.*;

public class MissingSum_260729{
    public int solution(int[] numbers) {
        int answer = 45;
        for(int i=0;i<numbers.length;i++){
            answer-=numbers[i];
        }
        return answer;
    }
    public static void main(String[] args) {
        Scanner s=new Scanner(System.in);
        System.out.print("배열을 입력하세요: ");
        String[] numbers = s.nextLine().split(" ");
        int[] numArray = new int[numbers.length];
        for (int i = 0; i < numbers.length; i++) {
            numArray[i] = Integer.parseInt(numbers[i]);
        }
       MissingSum_260729 sol = new MissingSum_260729();
        int result = sol.solution(numArray);
        System.out.println(result);
    }
}

File: algorithm/python/Q_260729/MissingSum_260729.py
#문제: 없는 숫자 더하기
#링크: (https://school.programmers.co.kr/learn/courses/30/lessons/86051?language=python3)
#날짜: 2026-07-29

def solution(numbers):
    answer = -1
    for i in range(10):
        if(i not in numbers):
            answer+=i
    if(answer != -1):
        answer+=1
    return answer

numbers=list(map(int,input("배열을 입력하세요:").split()))
print(solution(numbers))

#def solution(numbers):
# return 45-sum(numbers)